### PR TITLE
support passing time/size params for the scroller

### PR DESCRIPTION
**Pull request by mickeyn**
Adding support for using setting of scroller time/size values.

I want to make sure the wording in the doc explains that it's not recommended to use / you have to understand what these params do.

### CODE REVIEW

**Comment by mickeyn** [comment]
@oalders @reyjrar - can you please review this PR?

File: lib/MetaCPAN/Client.pm [hunk 1]
```diff
@@ -605,6 +605,30 @@ we can't to begin with (like non-leaf fields that hold a structure)
 
     my $module = $mcpan->all('releases', { _source => "stat" });
 
+=head2 scroller_time
+
+Note: please use with caution.
+
+This parameter will set the max. lifetime of the ElasticSearch scroller
```
Comment (oalders) [inline lib/MetaCPAN/Client.pm]: "This parameter will set the maximum lifetime of the Elasticsearch scroller"

**Comment by mickeyn** [comment]
@oalders cool, thanks.

**Comment by mickeyn** [comment]
test failure cannot be recreated... probably temp and doesn't seem to have anything to do with the changes.
going ahead with the merge/release.

**Pull request merged**